routes/folders.js

```diff
@@ -20,6 +24,37 @@ async function listFolderContent (req, res, next) {
     res.status(200).json({ folderPages })
 }
 
+// List pages and directories from all folders
+async function listAllFolderContent (req, res, next) {
+    const { accessToken } = req
+    const { siteName } = req.params
+
+    const IsomerDirectory = new Directory(accessToken, siteName)
+    const folderType = new RootType()
+    IsomerDirectory.setDirType(folderType)
+    const repoRootContent = await IsomerDirectory.list()
+
+    const allFolders = repoRootContent.reduce((acc, curr) => {
+        if (
+            curr.type === 'dir'
+            && !ISOMER_TEMPLATE_DIRS.includes(curr.name)
+            && curr.name.slice(0, 1) === '_'
+        ) acc.push(curr.path)
```

Comment:
We could remove the underscore on the backend instead of the frontend, but I'm open to either option

Reply:
Resolved with [7f8b635](https://github.com/isomerpages/isomercms-backend/pull/117/commits/7f8b63524bbe3695a65a539927b9e2d1442f0c85). For consistency, we will abstract the `_` character away from the frontend entirely, and the backend will be responsible for removing and appending the `_` character